src/main/plugins/lastfm.ts

```diff
@@ -169,6 +155,40 @@ export default class LastFMPlugin {
         }
     }
 
+    private async getPrimaryArtist (attributes: any) {
+        const songId = attributes.playParams.catalogId || attributes.playParams.id
+
+        if (!this._store.lastfm.enabledRemoveFeaturingArtists || !songId) return attributes.artistName;
+
+        const res = await this._win.webContents.executeJavaScript(`
+            (async () => {
+                const subMk = await MusicKit.getInstance().api.v3.music(`/v1/catalog/${MusicKit.getInstance().storefrontId}/songs/${songId}`, {
```

Comment:
This causes an error with Typescript. `TS2362: The left-hand side of an arithmetic operation must be of type 'any', 'number', 'bigint' or an enum type.`